languages/python/exercises/concept/bools/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+In this exercise, you need to translate some rules from the classic game Pac-Man into Elixir functions.
+
+You have four rules to translate, all related to the game states.
```

Comment:
I think Elixir is a remnant of the source exercise :) That said, I don't actually think the "into Elixir functions" bit is the best way to describe the exercise. Maybe do something like this:

```suggestion
In this exercise, you need to implement some of the rules from the classic game Pac-Man.

You have four rules to implement, all related to the game states.
```

Reply:
Yep, remnant